crates/driver/src/infra/observe/mod.rs

```diff
@@ -335,3 +339,8 @@ pub fn order_excluded_from_auction(
 ) {
     tracing::trace!(uid=?order.uid, ?reason, "order excluded from auction");
 }
+
+/// Observe that a settlement was simulated
+pub fn simulated(tx: &eth::Tx, gas: Gas) {
+    tracing::debug!(?tx, gas = ?gas.0, "simulated settlement");
```

Comment:
BTW, since you already brought up `trace` logs in the description:
The pattern of putting all logs into the `observe` module is not very compatible with selectively enabling traces which is one reason why I advocated strongly against this experiment.
IMO it's time to move all the logs out of `observe` and into the place where they actually get logged.
Doesn't have to happen in this PR of course; I just wanted to bring it up.